instant.ts

```diff
@@ -98,47 +99,88 @@ async function runTests(path: string) {
   }
 }
 
-const orderPackageIds = (allPackages, chosenPackageIds) => {
-  function resolveDeps(id, currentStack) {
-    if (currentStack.includes(id))
-      throw Error(`Circular dependency present for id ${id}`)
-    currentStack.push(id)
+export const createDependencyTree = (allPackages, chosenPackageIds) => {
```

Comment:
If a error is thrown here, I think it's not catched anywhere.

Reply:
It's caught here https://github.com/openhie/instant-v2/pull/68/files/975dd1bdf54ac83d3acfeb53379a44e358977a40#diff-6092f43171251fedb4b217c0b8c26ec4c108ba7d2f02512b8c4bdfb52928084fR440 and the process is terminated.